src/Illuminate/Database/Query/Builder.php

```diff
@@ -837,6 +850,12 @@ protected function invalidOperator($operator)
                ! in_array(strtolower($operator), $this->grammar->getOperators(), true);
     }
 
+    protected function isBinaryOperator($operator)
+    {
+        return in_array(strtolower($operator), $this->binaryOperators, true) ||
+               in_array(strtolower($operator), $this->grammar->getBinaryOperators(), true);
```

Comment:
What is your reason for calling `strtolower($operator)`? In my opinion the operator has now lower or upper case.

```suggestion
        return in_array($operator, $this->binaryOperators, true) ||
               in_array($operator, $this->grammar->getBinaryOperators(), true);
```

Reply:
None in particular, I copied what I saw in the `invalidOperator`.